perceval/utils/statevector.py

```diff
@@ -239,6 +239,22 @@ def m(self):
     def n_max(self):
         return self._n_max
 
+    @staticmethod
+    def tensor_product(svd1, svd2, prob_threshold: float = 0):
+        """
+        Compute the tensor product of two SVDistribution with an optional probability threshold
+        """
+        if len(svd1) == 0:
+            return svd2
+        new_dist = SVDistribution()
+        for sv1, proba1 in svd1.items():
+            for sv2, proba2 in svd2.items():
+                if proba1 * proba2 < prob_threshold:
+                    continue
+                sv = sv1 * sv2
+                new_dist[sv] += proba1 * proba2
```

Comment:
new_dist[sv] doesn't exists, no ? So you can't do += ? Or is it handle in the __getitem__method ?